From a8fd4b1f563d24d4296c3e8225c8404e2724d4c2 Mon Sep 17 00:00:00 2001
From: Jordan Christiansen <xordspar0@gmail.com>
Date: Sun, 15 Mar 2020 16:55:33 -0500
Subject: [PATCH] Remove redundant demoextend definition

GCC 10 enables -fno-common by default, which causes the linker to fail when
there are multple definitions of a global variable.

See https://gcc.gnu.org/gcc-10/porting_to.html

[Retrieved from:
https://github.com/chocolate-doom/chocolate-doom/commit/a8fd4b1f563d24d4296c3e8225c8404e2724d4c2]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 src/hexen/mn_menu.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/hexen/mn_menu.c b/src/hexen/mn_menu.c
index 059f45b3e..a97b7fcd1 100644
--- a/src/hexen/mn_menu.c
+++ b/src/hexen/mn_menu.c
@@ -131,7 +131,6 @@ boolean MenuActive;
 int InfoType;
 int messageson = true;
 boolean mn_SuicideConsole;
-boolean demoextend; // from h2def.h
 
 // PRIVATE DATA DEFINITIONS ------------------------------------------------
 
